core/server.ts

```diff
@@ -1,32 +1,27 @@
+/// <reference lib="deno.unstable" />
+// Deno.Server.shutdown() is unstable
+
 import { posix } from "../deps/path.ts";
 import Events from "./events.ts";
-import {
-  serveFile as HttpServeFile,
-  Server as HttpServer,
-} from "../deps/http.ts";
+import { serveFile as HttpServeFile } from "../deps/http.ts";
 
 import type { Event, EventListener, EventOptions } from "../core.ts";
-import type { ConnInfo } from "../deps/http.ts";
 
 /** The options to configure the local server */
 export interface Options {
   /** The root path */
   root: string;
-
-  /** The port to listen on */
-  port: number;
```

Comment:
I think server options should keep as they are.

Reply:
`Options` now extends `Deno.ServeOptions`.